js/ui/html_editor/modules/tableResizing.js

```diff
@@ -0,0 +1,491 @@
+import $ from '../../../core/renderer';
+import eventsEngine from '../../../events/core/events_engine';
+import { isDefined } from '../../../core/utils/type';
+import { addNamespace } from '../../../events/utils/index';
+import _windowResizeCallbacks from '../../../core/utils/resize_callbacks';
+import { move } from '../../../animation/translator';
+import { getBoundingRect } from '../../../core/utils/position';
+import BaseModule from './base';
+import Draggable from '../../draggable';
+import { each } from '../../../core/utils/iterator';
+import { getWindow, hasWindow } from '../../../core/utils/window';
+import { extend } from '../../../core/utils/extend';
+
+const DX_COLUMN_RESIZE_FRAME_CLASS = 'dx-table-resize-frame';
+const DX_COLUMN_RESIZER_CLASS = 'dx-htmleditor-column-resizer';
+const DX_ROW_RESIZER_CLASS = 'dx-htmleditor-row-resizer';
+const DEFAULT_MIN_COLUMN_WIDTH = 40;
+
+const TIMEOUT = 100;
+
+const DRAGGABLE_ELEMENT_OFFSET = 2;
+
+const MODULE_NAMESPACE = 'dxHtmlTableResizingModule';
+
+const POINTERDOWN_EVENT = addNamespace('dxpointerdown', MODULE_NAMESPACE);
+const SCROLL_EVENT = addNamespace('scroll', MODULE_NAMESPACE);
+
+export default class TableResizingModule extends BaseModule {
+    constructor(quill, options) {
+        super(quill, options);
+        this.enabled = !!options.enabled;
+        this._tableResizeFrames = [];
+        this._minColumnWidth = options.minColumnWidth ?? DEFAULT_MIN_COLUMN_WIDTH;
+        this._minRowHeight = options.minRowHeight ?? DEFAULT_MIN_COLUMN_WIDTH / 2;
+        this._quillContainer = this.editorInstance._getQuillContainer();
+        this._quillInstance = quill;
+
+        if(this.enabled) {
+            this._attachResizerTimeout = setTimeout(() => {
+
+                const $tables = this._findTables();
+                if($tables.length) {
+                    this._fixTablesWidths($tables);
+                    this._createResizeFrames($tables);
+                    this._updateFramesPositions();
+                    this._updateFramesSeparators();
+                }
+
+                this._attachEvents();
+            }, TIMEOUT);
+
+            this.addCleanCallback(this.clean.bind(this));
+            this._resizeHandler = _windowResizeCallbacks.add(this._resizeHandler.bind(this));
+        }
+    }
+
+    _attachEvents() {
+        eventsEngine.on(this.editorInstance._getContent(), SCROLL_EVENT, this._updateFramesPositions.bind(this));
+
+        this._quillTextChangeHandler = this._getQuillTextChangeHandler.bind(this);
+        this._quillInstance.on('text-change', this._quillTextChangeHandler);
+    }
+
+    _detachEvents() {
+        eventsEngine.off(this.editorInstance._getContent(), MODULE_NAMESPACE);
+        this._quillInstance.off('text-change', this._quillTextChangeHandler);
+    }
+
+    _getQuillTextChangeHandler(delta, oldContent, source) {
```

Comment:
Why `get`? This is a handler, not a factory or getter.

You could rework it:
```
_getHandler() {
 return () => (delta, oldContent, source) => {
    ...
  };
}
```
So you don't need to bind the context of the function with the bind method